vendor/profile.ps1

```diff
@@ -5,13 +5,33 @@ if( -not $env:PSModulePath.Contains($CmderModulePath) ){
     $env:PSModulePath = $env:PSModulePath.Insert(0, "$CmderModulePath;")
 }
 
+try {
+    Get-command -Name "git" -ErrorAction Stop >$null
+    Import-Module -Name "posh-git" -ErrorAction Stop >$null
+    $gitStatus = $true
+} catch {
+    Write-Warning "Missing git support"
+    $gitStatus = $false
+}
+
+function checkGit($Path) {
```

Comment:
@Jackbennett @MartiUK
I don't understand the reason of this function. Posh-Git checks himself what you write, isn't it?